src/db_impl.h

```diff
@@ -149,7 +155,9 @@ class TitanDBImpl : public TitanDB {
   Statistics* stats_;
 
   std::unordered_map<uint32_t, std::shared_ptr<TableFactory>>
-      original_table_factory_;
+      base_table_factory_;
+  std::unordered_map<uint32_t, std::shared_ptr<TitanTableFactory>>
+      titan_table_factory_;
```

Comment:
Do we update these two map after dropping a CF? I don't see related logic.

Reply:
adding them now.